metrics/examples/prometheus/kubernetes.yaml

```diff
@@ -436,6 +439,33 @@ data:
         replacement: $1
         action: replace
 
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-rules
```

Comment:
These are just rules for the alert manager, so I wonder if we should reflect that in the name.

Reply:
Technically it could contain recording rules as well but I haven't tried if having both types in the same file works. In any case we are not providing recording rules example.

Reply:
What are recording rules?

Reply:
Prometheus can precompute stuff before saving data into the internal storage so instead of recording the scraped metrics it can record computed ones. More info here: sum(kafka_server_replicamanager_underreplicatedpartitions).
Btw because we are working on alerts here, I renamed it just "alerting-rules".

Reply:
Actually I didn't change the name because this ConfigMap is mounted as volume in the prometheus deployment for getting the rules. Because of the other possible rules (recording) I wuold leave as it is for now.